cosmo_tester/framework/test_hosts.py

```diff
@@ -522,13 +522,8 @@ def _create_config_file(self, upload_license=True):
     def bootstrap(self, enter_sanity_mode=True, upload_license=False,
                   blocking=True, restservice_expected=True):
         self.restservice_expected = restservice_expected
-        manager_install_rpm = \
-            ATTRIBUTES.cloudify_manager_install_rpm_url.strip() or \
-            util.get_manager_install_rpm_url()
```

Comment:
Do we use get_manager_install_rpm_url anywhere any more? If not, let's splat it.

Reply:
Manager of manager test use it and cannot drop it since need to download the rpm to the central manager image and drop the rpm to specific path so that spire plugin can use it

Reply:
https://github.com/cloudify-cosmo/cloudify-system-tests/blob/master/cosmo_tester/test_suites/manager_of_managers/mom_test.py#L103